cinema/views.py

```diff
@@ -158,6 +174,21 @@
 
         return queryset
```

Comment:
Similar to the `get_queryset` method in `MovieViewSet`, the `get_queryset` method in `MovieSessionViewSet` could be broken down into smaller methods for better readability and maintainability.